src/schemas/Object.js

```diff
@@ -17,7 +17,21 @@ export const denormalize = (schema, input, unvisit, entities) => {
   Object.keys(schema).forEach((key) => {
     const localSchema = schema[key];
     if (object[key]) {
-      object[key] = unvisit(object[key], localSchema, entities);
+      if (Array.isArray(object[key])) {
+        object[key] = unvisit(object[key], localSchema, entities);
+      } else {
+        const skey = localSchema.key;
```

Comment:
What does `skey` stand for? No need to shorten variable names and sacrifice clarity.